server/routes.js

```diff
@@ -25,6 +26,29 @@ function applyContentSecurityPolicy(res) {
 
 exports.routes = function () {
   return {
+    lbheartbeat: function(req, resp) { resp.send(200); },
+    version: function(req, resp) {
+      // look for a version.json file in the top level code directory
+      // if it exists, read / return it, otherwise return nothing
+      var verfile = __dirname + '../../version.json';
+      fs.realpath(verfile, function(err, resolvedPath) {
+        if (err) {
+          resp.send(404);
+          return;
+        }
+
+        fs.readFile(resolvedPath, function(err, data) {
+          if (err) {
+            resp.send(503);
+            return;
+          }
+
+          resp.setHeader('Content-Type', 'application/json');
+          resp.send(data);
+        });
+      });
+    },
+
```

Comment:
One small note: in other apps, in production, we use `require` to try to load the version.json, and cache it in memory, since it won't change. But probably not a big deal for moz-idp.